dll/win32/devmgr/lang/cs-CZ.rc

```diff
@@ -313,6 +313,7 @@ BEGIN
   IDS_HINT_UNINSTALL    " Odinstaluje ovladače vybraného zařízení."
   IDS_HINT_ADD          " Umožní přidat do systému zastaralé zařízení nepodporující Plug and Play."
   IDS_HINT_ABOUT        " Ukončí program.."
+  IDS_HINT_EXIT         " Beendet das Programm."
```

Comment:
This should probably be english instead of german?

Reply:
Err, yes.